$NetBSD: patch-aa,v 1.3 2016/10/01 13:00:10 joerg Exp $

Don't creat this file for pkgsrc.
Call versioned 2to3 binary.

--- setup.py.orig	2013-04-17 09:38:31.000000000 +0000
+++ setup.py
@@ -109,14 +109,6 @@ class MyInstallLib(install_lib.install_l
     def run(self):
         """overridden from install_lib class"""
         install_lib.install_lib.run(self)
-        # create Products.__init__.py if needed
-        if subpackage_of:
-            product_init = join(self.install_dir, subpackage_of, '__init__.py')
-            if not exists(product_init):
-                self.announce('creating %s' % product_init)
-                stream = open(product_init, 'w')
-                stream.write(EMPTY_FILE)
-                stream.close()
 
 
 class MyBuildPy(build_py):
@@ -140,7 +132,7 @@ class MyBuildPy(build_py):
                     # process manually python file in include_dirs (test data)
                     from subprocess import check_call
                     print('running 2to3 on', dest) # parens are NOT optional here for py3k compat
-                    check_call(['2to3', '-wn', dest])
+                    check_call(['2to3-' + os.environ['PYVERSSUFFIX'], '-wn', dest])
 
 def install(**kwargs):
     """setup entry point"""
